src/main/java/com/conveyal/taui/models/AnalysisRequest.java

```diff
@@ -138,14 +138,11 @@ public AnalysisTask populateTask (AnalysisTask task, Project project) {
 
         task.monteCarloDraws = monteCarloDraws;
 
-        if (travelTimePercentile == null) {
+        if (task.getType() == AnalysisTask.Type.TRAVEL_TIME_SURFACE) {
```

Comment:
Do we want to overwrite the percentile values for all travel-time surface requests here?

If we change https://github.com/conveyal/analysis-ui/blob/3ac86c9f3d816656df00f9ed0ba75faef3d4993f/lib/constants/analysis.js#L3 in the future, will this line need to be changed too?

If I manually edited a single-point request to request different percentiles, will those be ignored?

Reply:
Previously we were always sending `null` for `TRAVEL_TIME_SURFACE` requests but we are now saving that value in the same profile request object we send over so it's needs to be overridden somewhere. We could set the "percentiles" manually on the front end based off of the saved "travelTimePercentile" value if it's a regional analysis and the default array if it's a `TRAVEL_TIME_SURFACE` request.